content/en/agent/vector_aggregation.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Aggregating Multiple Agents Using Vector
+kind: documentation
+further_reading:
+- link: "/logs/"
+  tag: "Documentation"
+  text: "Collect your logs"
+- link: "/agent/proxy/"
+  tag: "Documentation"
+  text: "Configure the Agent to use a proxy"
+
+---
+
+## Overview 
+
+The Datadog Agent can be used along with [Vector][1]. In this scenario Agents will send data
+to Vector which will aggregate data from multiple upstream Agents:
+
+`Agents ---> Vector ---> Datadog`
+
+This scenario differs from using a [proxy][2] since Vector is able to process data before sending
+it to Datadog and other destinations. Vector capabilities include, among others:
+others:
+
+* [Log parsing, structuring, and enrichment][3]
+* Cost reduction through [sampling][4]
+* [Redaction of PII][5] and sensitive data before it leaves your network
+* [Deriving metrics from logs][6] for cost effective analysis
+* [Routing to multiple destinations][7]
+
+**Note**: Only logs aggregation is currently supported.
+
+**Note**: Vector can also directly collect logs from alternative sources. If doing so, it is likely that 
+third party logs won't come with proper tagging. A convenient way to [add tags][8], source or service
+values is to use the [Vector Remap Language][3].
+
+## Configuration
+
+### Agent configuration
+To send logs to Vector, update the Agent configuration file, `datadog.yaml`.
+Only the logs data type is currently supported. Update the following values in the `datadog.yaml` file:
+```yaml
+logs_config:
+  logs_dd_url: "<VECTOR_HOST>:<VECTOR_PORT>"
```

Comment:
It might be worth clarifying that `VECTOR_PORT` specifically means the port used in the `datadog_logs` source (the example below uses `8080`), not a generic "Vector port".

Reply:
I update the doc to add a bit more detail about that.